src/components/location-modal/index.js

```diff
@@ -1,12 +1,14 @@
 import { connect } from 'react-redux';
 import { currentLocation } from 'modules/locations/selectors';
+import { conservationHotspots } from 'modules/widgets/selectors';
 import { closeSearchPanel } from 'modules/locations/actions';
 import Component from './component';
 
 const mapStateToProps = state => ({
   isOpened: state.locations.isOpened,
   currentLocation: currentLocation(state),
-  locations: state.locations.list,
+  data: conservationHotspots(state),
```

Comment:
Data is very generic. In this case, I prefer to use a label like `conservationHotspots`